### PR TITLE
swg release 0.1.22.146

### PR DESCRIPTION
**Pull request by cvolzke4**
swg release 0.1.22.146

### CODE REVIEW

File: third_party/subscriptions-project/config.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ class Doc {
 
   /**
    * The `Document` node or analog.
-   * @return {!Node}
+   * @return {!Document}
```
Comment (jpettitt) [inline third_party/subscriptions-project/config.js:123]: Not sure if this is going to break PWA's

Comment (cvolzke4) [reply third_party/subscriptions-project/config.js:123]: The type annotation? How can I check

Comment (jpettitt) [reply third_party/subscriptions-project/config.js:123]: @qidonna this was your change doc.js must match the one in AMP so changing the type here could be an issue.  The amp version is here https://github.com/ampproject/amphtml/blob/06123ab062b2be8d04bdb3e232c7c02a8cffe09b/extensions/amp-subscriptions/0.1/doc-impl.js#L48

**Review by jpettitt — CHANGES_REQUESTED** [review]

File: third_party/subscriptions-project/config.js (hunk 1 above)
Comment (stellachui) [reply third_party/subscriptions-project/config.js:123]: Reverted in https://github.com/subscriptions-project/swg-js/pull/1501. @cvolzke4 could you help cherrypick into the release?

Comment (cvolzke4) [reply third_party/subscriptions-project/config.js]: I've applied the revert to this PR.

Comment (cvolzke4) [reply third_party/subscriptions-project/config.js]: jpettitt@ PTAL

**Comment by cvolzke4** [comment]
Superceded by:
https://github.com/ampproject/amphtml/pull/32758

**Pull request closed without merging**